src/qcodes/instrument_drivers/Keithley/Keithley_2450.py

```diff
@@ -485,7 +485,7 @@ def __init__(self, parent: "Keithley2450", name: str, proper_function: str) -> N
 
     def _set_proper_function(self, value: float) -> None:
         self.write(f"SOUR:{self._proper_function} {value}")
-        if self.block_during_ramp():
+        if self.block_during_ramp:
```

Comment:
@aaronsharpe I am a bit confused by this. `block_during_ramp` is a QCoDeS parameter. This means that it is callable. Calling it without arguments is equivalent to calling `.get()` on the parameter. With your change this would test an instance of the parameter class as a bool which will always evaluate to True.


```

In [1]: from qcodes.parameters import Parameter
In [2]: a = Parameter("a")
In [3]: bool(a)
Out[3]: True
```

Reply:
Not sure what happened. On one of our setups, it was throwing an error but not on my test rig. Will delete pull request pending what I find